15_tictactoe/go/grid/grid.go

```diff
@@ -1,10 +1,26 @@
 package grid
 
+// Define column and row
 type Grid struct {
-	// TODO
+	Cells []int
 }
 
+// testGrid := Grid {
+// 	Cells: []int{3,3},
+// }
+
 func (g Grid) String() string {
 	// TODO
+	// |||____
+	// |__|__|__|
+	// |__|__|__|
+	// |__|__|__|
+	// row := g.Cells * "__"
+	// column := g.Cells
+	Cells := []string{}
```

Comment:
you can do this in a single operation, using https://pkg.go.dev/fmt#Sprintf

The function should look like

```go
func (g Grid) String() string {
  return fmt.Sprintf(
    "<TODO>",
    args...
  )
}
```